src/sqlancer/clickhouse/ClickHouseErrors.java

```diff
@@ -1,85 +1,65 @@
+                "Cannot parse Int32 from String, because value is too short", "Cannot parse NaN.: while converting", // https://github.com/ClickHouse/ClickHouse/issues/22710
+                "Cannot parse infinity.", "Cannot parse number with a sign character but without any numeric character",
+                "Cannot parse number with multiple sign (+/-) characters or intermediate sign character",
+                "Cannot parse string", "Cannot read floating point value",
+                "Cyclic aliases: default expression and column type are incompatible", "Directory for table data",
+                "Directory not empty", "Expected one of: compound identifier, identifier, list of elements (version", // VALUES
+                                                                                                                      // ()
+                "Function 'like' doesn't support search with non-constant needles in constant haystack", "Illegal type",
+                "Illegal value (aggregate function) for positional argument in GROUP BY",
+                "Invalid escape sequence at the end of LIKE pattern", "Invalid type for filter in", "Memory limit",
+                "OptimizedRegularExpression: cannot compile re2", "Partition key cannot contain constants",
+                "Positional argument out of bounds", "Sampling expression must be present in the primary key",
+                "Sorting key cannot contain constants", "There is no supertype for types", "argument of function",
+                "but its arguments considered equal according to constraints", "does not return a value of type UInt8",
+                "doesn't exist", // TODO: consecutive test runs can lead to dropped database
+                "in block. There are only columns:", // https://github.com/ClickHouse/ClickHouse/issues/42399
+                "invalid character class range", "invalid escape sequence",
+                "is not under aggregate function and not in GROUP BY", "is not under aggregate function",
+                "is violated at row 1. Expression:", // TODO: check constraint on table creation
+                "is violated, because it is a constant expression returning 0. It is most likely an error in table definition",
+                "there are only columns", "there are columns", "in block. (NOT_FOUND_COLUMN_IN_BLOCK)",
+                "Missing columns", "Ambiguous column",
+                "Must be one unsigned integer type. (ILLEGAL_TYPE_OF_COLUMN_FOR_FILTER)",
+                "Floating point partition key is not supported", "Cannot get JOIN keys from JOIN ON section",
+                "ILLEGAL_DIVISION", "DECIMAL_OVERFLOW",
+                "Cannot convert out of range floating point value to integer type",
+                "Unexpected inf or nan to integer conversion", "No such name in Block::erase", // https://github.com/ClickHouse/ClickHouse/issues/42769
```

Comment:
Just noticed. Some of these are known bugs, for which we should the pattern explained at https://github.com/sqlancer/sqlancer/blob/main/CONTRIBUTING.md#unfixed-bugs (e.g., see the Citus implementation). I guess it's not in scope of the PR, but if you are interested, @malwaregarry, you could address those (perhaps some of them have also been fixed by now).

Reply:
thanks for pointing this out. I can work on this after I'm done with NoREC